certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+        },
+        "secondaryName": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "relationship": {
+          "description": "Gets or Sets Relationship",
+          "type": "string"
+        },
+        "currentAddress": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderAddress"
+        },
+        "forwardingAddress": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderAddress"
+        },
+        "communications": {
+          "description": "Gets or Sets Communications",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Communication"
+          }
+        },
+        "realtor": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "attorney": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "tax1099": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Irs1099"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TransactionParties": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "sellers": {
+          "description": "Gets or Sets Sellers",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Seller"
+          }
+        },
+        "buyers": {
+          "description": "Gets or Sets Buyers",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Buyer"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TransactionType": {
+      "format": "int32",
+      "description": "Gets or Sets TransactionType",
+      "enum": [
+        0,
+        1,
+        2
+      ],
+      "type": "integer"
+    },
+    "PubCS.TitleOrder.Api.Models.LoanPayment": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "principal": {
+          "format": "double",
+          "description": "Gets or Sets Principal",
+          "type": "number"
+        },
+        "interest": {
+          "format": "double",
+          "description": "Gets or Sets Interest",
+          "type": "number"
+        },
+        "mortgageInsurance": {
+          "format": "double",
+          "description": "Gets or Sets MortgageInsurance",
+          "type": "number"
+        },
+        "taxes": {
+          "format": "double",
+          "description": "Gets or Sets Taxes",
+          "type": "number"
+        },
+        "hoa": {
+          "format": "double",
+          "description": "Gets or Sets Hoa",
+          "type": "number"
+        },
+        "totalMonthly": {
+          "format": "double",
+          "description": "Gets or Sets TotalMonthly",
+          "type": "number"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.LoanDetail": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.